util/json.go.tmpl

```diff
@@ -0,0 +1,57 @@
+{{/*
+    JSON Converter CC (JSONify)
+    
+    Usage: run the json command to find out :)
+
+    Recommended Trigger Type: Command
+    Recommended Trigger: json
+    
+    Credits:
+    Devonte <https://github.com/devnote-dev>
+*/}}
+
+{{$a := parseArgs 2 "```json <Channel:Mention> <Message:ID>\njson <Channel:Name> <Message:ID>\njson <Channel:ID> <Message:ID>\njson 0 <Message:ID>\n```**Optional Flags:**\n```\n[-j strict-format:flag]\n[-file/f attachment:flag]```"
+(carg "string" "chan") (carg "int" "msgID") (carg "string" "flag: -j | -f")}}
+{{$type := dict 0 "Default Message" 6 "Pinned Message" 7 "Join Message" 8 "Boost Message" 9 "Teir 1 Boost Message" 10 "Teir 2 Boost Message" 11 "Teir 3 Boost Message" 12 "Followed Channel Message" 19 "Reply Message"}}
+{{$struct := "Unknown"}}{{$ver := "JSONify v4.15"}}{{$fa := false}}{{$chan := .Channel}}
+{{$link := joinStr "/" "https://discordapp.com/channels" .Guild.ID .Channel.ID ($a.Get 1)}}{{$ctx := ""}}
+
+{{$l := sendMessageRetID nil (cembed "description" "Converting Message... <a:loading:760219029620523008>")}}
+{{if ne ($a.Get 0) "0"}}
+    {{$chan = getChannel (or (reFind `\d+` ($a.Get 0)) ($a.Get 0))}}
+    {{$link = joinStr "/" "https://discordapp.com/channels" .Guild.ID $chan.ID ($a.Get 1)}}
+{{end}}
+{{if ($msg := getMessage $chan.ID ($a.Get 1))}}
+    {{with $msg.Embeds}}{{$struct = print (title (index . 0).Type) " Embed"}}{{else}}
+    {{$struct = or (and $msg.Attachments "Attachment Message") (and (eq $msg.Type 0 6 7 8 9 10 11 12 19) ($type.Get (toInt $msg.Type))) $struct}}
```

Comment:
I don't think this `eq` check is necessary, since `$type.Get` will be false if the type msg is none of those.